validator/rpc/handle_slashing.go

```diff
@@ -0,0 +1,84 @@
+package rpc
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+	http2 "github.com/prysmaticlabs/prysm/v4/network/http"
+	slashing "github.com/prysmaticlabs/prysm/v4/validator/slashing-protection-history"
+	"go.opencensus.io/trace"
+)
+
+// ExportSlashingProtection handles the rpc call returning the json slashing history.
+// The format of the export follows the EIP-3076 standard which makes it
+// easy to migrate machines or Ethereum consensus clients.
+//
+// Steps:
+//  1. Call the function which exports the data from
+//     the validator's db into an EIP standard slashing protection format.
+//  2. Format and send JSON in the response.
+func (s *Server) ExportSlashingProtection(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "validator.ExportSlashingProtection")
+	defer span.End()
+
+	if s.valDB == nil {
+		http2.HandleError(w, "err finding validator database at path", http.StatusInternalServerError)
```

Comment:
```suggestion
		http2.HandleError(w, "could not find validator database", http.StatusInternalServerError)
```
I changed the error message to something more standard. I don't think we ever return `err` in an error message and the `path` word is unclear. What path?